src/assignment/modals/AddBlock.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { FunctionComponent, ReactNode, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+	BlockHeading,
+	Button,
+	Icon,
+	Modal,
+	ModalBody,
+	ModalProps,
+	Spacer,
+} from '@viaa/avo2-components';
+import { IconNameSchema } from '@viaa/avo2-components/dist/esm/components/Icon/Icon.types';
+
+import { EDIT_ASSIGNMENT_BLOCK_ICONS } from '../assignment.const';
+import { AssignmentBlockType, AssignmentFormState } from '../assignment.types';
+
+import './AddBlock.scss';
+
+type AddBlockType = AssignmentBlockType | 'COLLECTIE';
+
+interface AddBlockProps extends Pick<ModalProps, 'isOpen' | 'onClose'> {
+	assignment: AssignmentFormState;
+	onConfirm?: (type: AddBlockType) => void;
+}
+
+const AddBlock: FunctionComponent<AddBlockProps> = ({ assignment, isOpen, onClose, onConfirm }) => {
```

Comment:
```suggestion
const AddBlockModal: FunctionComponent<AddBlockProps> = ({ assignment, isOpen, onClose, onConfirm }) => {
```

Reply:
Het volgt de bestaande naming conventie in de repo niet. Alle modals hebben de suffix Modal: 
![image](https://user-images.githubusercontent.com/1710840/176399944-772abafc-8cd2-4d29-8c43-72806f271acd.png)